vendor/engines/split_accounts/config/locales/en.yml

```diff
@@ -9,6 +9,7 @@ en:
               percent_total: percent total must equal 100
               only_one_extra_penny: must have one split with extra penny
               duplicate_subaccounts: must have unique subaccounts
+              more_than_one_split: must have two or more
```

Comment:
This message becomes a little awkward when it displays: "Splits must have two or more". Maybe just add "subaccounts" to the end?